docs/sources/flow/reference/components/prometheus.integration.process_exporter.md

```diff
@@ -0,0 +1,108 @@
+The `prometheus.integration.process_exporter` component embeds
+[process_exporter](https://github.com/ncabatoff/process-exporter) for collecting process stats mined from `/proc`.
+
+## Usage
+
+```river
+prometheus.integration.process_exporter "LABEL" {
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`procfs_path`       | `string`                 | procfs mountpoint. | `/proc` | no
+`track_children`    | `bool`                   | If a proc is tracked, also track any of it's children that are not part of their own group. | `true` | no
+`track_threads`     | `bool`                   | Report on per-threadname metrics as well. | `true` | no
+`gather_smaps`      | `bool`                   | Gather metrics from the smaps file, which contains proportional resident memory size. | `true` | no
+`recheck_on_scrape` | `bool`                   | Recheck process names on each scrape. | `true` | no
+
+## Blocks
+The following blocks are supported inside the definition of `prometheus.integration.process_exporter`:
+
+Hierarchy        | Block      | Description | Required
+---------------- | ---------- | ----------- | --------
+process_names          | [process_names][]  | A collection of matching rules to use for deciding which processes to monitor. Each config can match multiple processes, which will be tracked as a single process "group." | no
+
+[process_names]: #process_names-block
+
+### process_names block
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`name`       | `string`                         | The name to use for identifying the process group name in the metric. By default, it uses the base path of the executable. (See below for template variable details). | `{{.ExeBase}}` | no
+`comm`       | `list(string)`                   | A list of strings that match the base executable name for a process, truncated to 15 characters. It is derived from reading the second field of `/proc/<pid>/stat`, stripped of parens. | | no
+`exe`        | `list(string)`                   | A list of strings that match `argv[0]` for a process. If there are no slashes, only the basename of `argv[0]` needs to match. Otherwise, the name must be an exact match. For example, "postgres" may match any postgres binary, but `/usr/local/bin/postgres` will only match a postgres process with that exact path. If any of the strings match, the process will be tracked. | | no
+`cmdline`    | `list(string)`                   | A list of regular expressions applied to the `argv` of the process. Each regex here must match the corresponding argv for the process to be tracked. The first element that is matched is `argv[1]`. Regex captures are added to the .Matches map for use in the name. | | no
```

Comment:
The descriptions here are quite long. Can you use shorter sentences and move more detailed explanations to outside of the table?

Reply:
Will do!